### DISCUSSION TITLE
cds-control does not display the custom component

### DISCUSSION THREAD

**Discussion by wendellestradairely** [vmware/clarity#6346] [Q&A] [answered]
I'm trying to use ng-select to a cds-control for custom components but it doesn't display the component:
```html
<cds-control>
	<label>Select an Option</label>
	<ng-select>
		<ng-option [value]="'male'">Male</ng-option>
		<ng-option [value]="'female'">Female</ng-option>
	</ng-select>
</cds-control>
```
![image](https://user-images.githubusercontent.com/50942719/132096721-9b44000e-0a2a-43a7-8d8b-7bb58711f42a.png)

**Comment by bbogdanov** [comment]
Hi @wendellestradairely,

Could you please provide a [Stackblitz](https://stackblitz.com/edit/node-who5wc) example?

**Comment by coryrylan** [comment, marked as answer]
This is because the `cds-control` attribute is needed to identify the custom component replacing the input. Checkout the "Generic content" section [here](https://clarity.design/storybook/core/?path=/story/forms-control--page).

```html
<cds-control>
  <label>Select an Option</label>
  <ng-select cds-control>
    <ng-option [value]="'male'">Male</ng-option>
    <ng-option [value]="'female'">Female</ng-option>
  </ng-select>
</cds-control>
```

**Comment by wendellestradairely** [comment]
@bbogdanov Sorry, I've been busy in the past few days. I could not replicate it in StackBlitz. I'm still using a combination of Core & Angular components.
@coryrylan The `cds-control` directive/attribute worked perfectly. 
Thanks guys!
